schema/artist/index.js

```diff
@@ -149,11 +149,15 @@ const ArtistType = new GraphQLObjectType({
       },
       formatted_nationality_and_birthday: {
         type: GraphQLString,
-        description: 'A string of the form "Nationality, Birthday"',
-        resolve: ({ birthday, nationality }) => {
+        description: 'A string of the form "Nationality, Birthday (or Birthday-Deathday)"',
+        resolve: ({ birthday, nationality, deathday }) => {
           let formatted_bday = (!isNaN(birthday) && birthday) ? 'b. ' + birthday : birthday;
           formatted_bday = formatted_bday && formatted_bday.replace(/born/i, 'b.');
 
+          if ((!isNaN(deathday) && deathday)) {
+            formatted_bday = `${formatted_bday.replace('b. ', '')}–${deathday.match(/\d+/)}`;
```

Comment:
Feel like this variable could be renamed now that it not only holds the birthday.